test/unit/org/apache/cassandra/io/sstable/SSTableReaderTest.java

```diff
@@ -1435,4 +1470,9 @@ private ColumnFamilyStore discardSSTables(String ks, String cf)
         cfs.discardSSTables(System.currentTimeMillis());
         return cfs;
     }
+
+    private void trackReleaseableRef(Ref<?> ref)
```

Comment:
Slightly pedantic, but return the ref and then invoke track releasable ref around the thing that actually allocates the ref so it never "escapes".

Optional though since the existing thing also works fines.

Reply:
Done - I like it.